lib/cosmos/tools/cmd_tlm_server/api.rb

```diff
@@ -89,6 +92,7 @@ def initialize
         'get_router_info',
         'get_cmd_cnt',
         'get_tlm_cnt',
+        'get_packet_loggers',
```

Comment:
I added this as well because you need it to effectively use get_packet_logger_info

Reply:
These should all probably be added to script as well.

Reply:
They are implemented in cosmos/script/cmd_tlm_server.rb